test/e2e/istio_test.go

```diff
@@ -111,8 +111,12 @@ func TestIstioWithKongIngressGateway(t *testing.T) {
 		return ready
 	}, time.Minute, time.Second)
 
+	t.Log("preparing the environment to run the controller manager")
```

Comment:
```suggestion
	t.Log("preparing the environment to run the controller manager")
```
We decided to capitalize, but we're not doing it for logs in tests, it's not so important though

Reply:
```suggestion
	t.Log("Preparing the environment to run the controller manager")
```